### PR TITLE
[DOCS] Expands hyperparameters conceptual docs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR expands the conceptual docs of hyperparameters about how the importance of optimized hyperparameters is calculated.

### Preview

[Calculating the importance of hyperparameters](https://stack-docs_1522.docs-preview.app.elstc.co/guide/en/machine-learning/master/hyperparameters.html#hyperparameter-importance)

### CODE REVIEW

**Review by valeriy42 — APPROVED** [review]
LGTM

File: docs/en/stack/ml/df-analytics/hyperparameters.asciidoc [hunk 1]
```diff
@@ -20,9 +20,22 @@ a model, and calculate its performance on validation data.
...
-see the specific type of validation loss (such as mean squared error or binomial
-cross entropy) that was used to compare each round of optimization. 
+{ref}/get-trained-models.html[get trained models API]. You can also see the 
+specific type of validation loss (such as mean squared error or binomial cross 
+entropy) that was used to compare each round of optimization using the 
+{ref}/get-dfanalytics-stats.html[get {dfanalytics-job} stats API].
+
+
+[[hyperparameter-importance]]
+== Calculating the importance of hyperparameters
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/hyperparameters.asciidoc:30]: > == Calculating the importance of hyperparameters

Do we need to link to this specific content from elsewhere? If not, I think this paragraph can be interwoven with the existing ones.

File: docs/en/stack/ml/df-analytics/hyperparameters.asciidoc [hunk 2]
```diff
@@ -20,9 +20,22 @@ a model, and calculate its performance on validation data.
...
+
+Different hyperparameters may affect the model accuracy to a different degree.
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/hyperparameters.asciidoc:32]: > Different hyperparameters may affect the model accuracy to a different degree. 

I think "accuracy" is one of those loaded terms in data science. Is it the right word here or should we use something like "job results" or "analysis"?

File: docs/en/stack/ml/df-analytics/hyperparameters.asciidoc [hunk 3]
```diff
@@ -20,9 +20,22 @@ a model, and calculate its performance on validation data.
...
+
+
+[[hyperparameter-importance]]
+== Calculating the importance of hyperparameters
+
+Different hyperparameters may affect the model accuracy to a different degree. 
+To estimate the importance of the optimized hyperparameters, analysis of 
+variance decomposition is used. The resulting `absolute importance` shows how 
+much the variation of a hyperparameter impacts the variation in the validation 
+loss. Additionally, `relative importance` is also computed which gives the
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/hyperparameters.asciidoc:36]: > Additionally, `relative importance` is also computed which gives the 

It's not clear to me from reading this where I find these `absolute importance` and `relative importance` values. If you view these details in Kibana or in API results, it likely works to merge this detail into the preceding paragraph.

**Pull request merged**